webapp/src/pages/ResponsePage/ResponsePage.js

```diff
@@ -50,15 +50,24 @@ class ResponsePage extends Component {
                 applicationForm: responses[1].formSpec,
                 applicationData: responses[2].detail,
                 tagList: responses[3].tags,
-                reviewers: responses[4].reviewers,
-                error: responses[0].error || responses[1].error || responses[2].error || responses[3].error || responses[4].error,
+                error: responses[0].error || responses[1].error || responses[2].error || responses[3].error,
             }, () => {
-                this.handleData(); 
                 this.getOutcome();
+                this.getReviewResponses(responses[2].detail);
             });
         });
     };
 
+    getReviewResponses(applicationData) {
+        reviewService.getResponseReviewAdmin(applicationData.id, this.props.event.id)
+            .then(resp => {
+                this.setState( {
+                    reviewResponses: resp.form.review_responses,
```

Comment:
What happens if there is no review form defined for an event and the API raises a review_form_not_found error? I think it will work and silently ignore the error (which I guess is OK), but just wanted to double check if you've tested that the response page still works in that scenario?

Reply:
I didn't accounted for this, but good catch! Thanks!